src/apply.ts

```diff
@@ -227,23 +115,45 @@ export async function apply(options: ReadOptions): Promise<number> {
 		);
 	}
 
+	type RemoteImage = {
+		registry: string;
+		repository: string;
+		image_name: string;
+		content_hash: string;
+		service: string;
+		status: string;
+	};
```

Comment:
What is RemoteImage and why do you need it? I don’t see code that uses anything else besides “service”.

Reply:
I was only able to use the index for `NavigationResource<Image>` when using a map rather than looping through the release_images.  There is probably another approach but will take me more time to discover with the same result.

Reply:
We already changed the approach over a call.  The changes have been pushed to the branch.